### DISCUSSION TITLE
set the projectreference properly

### DISCUSSION THREAD

**Discussion by ghost** [dotnet/templating#6157] [Template authoring] [answered]
Hi,
I want to generate a template.json file for a project which has a reference to multiple projects in our repository, the challenge I have is, the projectreference is constant and if a user make a project in subfolder, it doesn't find referenced projects.
For example I have the following ProjectReference
<ProjectReference Include="..\..\..\..\..\base\SomeProj\src\AspNetCore\SomeProj.csproj" />
so how I can make the Include path dynamically, so if a user generate the template in another level of directory (but under the same repository) Visual Studio be happy about that?

**Comment by YuliiaKovalova** [comment, marked as answer]
Hi @vashdigital,

your example of project reference was missed. Could you add it again?
did you consider using [post-actions](https://github.com/dotnet/templating/wiki/Post-Action-Registry) for this goal?

**Reply by ghost** [reply]
actually by using post-actions it has been resolved, thnx
